idl/chromadb/proto/logservice.proto

```diff
@@ -3,6 +3,17 @@ syntax = "proto3";
 package chroma;
 option go_package = "github.com/chroma/chroma-coordinator/internal/proto/logservicepb";
 
-service LogService {
+import "chromadb/proto/chroma.proto";
+
+message PushLogsRequest {
+  string collection_id = 1;
+  repeated SubmitEmbeddingRecord records = 2;
+}
 
+message PushLogsResponse {
```

Comment:
This should have error types propagated in the response.

Reply:
does grpc messages not have error types?

Reply:
I can try to write  a test to verify

Reply:
It does, but we should expand and add message/status etc

https://cloud.google.com/apis/design/errors#error_model

Reply:
https://github.com/chroma-core/chroma/blob/887d0b54a314ee0ce1d4c441f7eb4ab409433234/idl/chromadb/proto/chroma.proto#L7

Reply:
https://grpc.io/docs/guides/error/

Reply:
I'm returning the error status from the record_log_service? what is the change requested?

Reply:
Follow the status pattern used in the rest of coordinator.proto.

Reply:
https://github.com/chroma-core/chroma/blob/887d0b54a314ee0ce1d4c441f7eb4ab409433234/idl/chromadb/proto/coordinator.proto#L35

Reply:
I don't think that is the correct pattern. I think we need to fix that code

Reply:
Either we can:

1. Implement the correct change uniformly
2. Follow the existing pattern

Fragmentation of standards is not something we should be ok with rn.

Reply:
I cannot fix everything I see. We should always be mindful about making incremental improvements. These improvements will come by priorities not by the fact that we see them. Also, fixing existing SysDB code is needed before we launch the product. I'm sure that we will fix it soon.